### PR TITLE
Port apigility.org to new website

### PR DESCRIPTION
**Pull request by weierophinney**
This patch ports the apigility.org website to api-tools.getlaminas.org,
and ensures that it has the look-and-feel established by the other
sites. It also:

- Migrates the codebase to Laminas MVC
- Updates all dependencies to stable versions, including the CommonMark toolchain
- Removes the installer script
- Refactors slightly to use class-based factories over Module-based ones
- Reworks the toolchain for retrieving releases to use tags

### CODE REVIEW

File: .deploy/deploy.sh [hunk 1]
```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+function log_error_and_exit () {
+    local LAST_COMMAND=$1
+    local LAST_STATUS=$2
+    logger "Error running $0; received ${LAST_STATUS} from ${LAST_COMMAND}"
+    exit ${LAST_STATUS}
+}
+
+echo "[BEGIN] deploy apigility.org"
```
Comment (michalbundyra) [inline .deploy/deploy.sh]: I believe we want to update here - no more apigility.org

File: .deploy/deploy.sh [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    COMPOSER_HOME=/var/cache/composer composer install --quiet --no-ansi --no-dev --no-interaction --no-progress --no-scripts --no-plugins --optimize-autoloader ;
+    if [ $? != 0 ];then log_error_and_exit composer_install $?; fi ;
+
+    # Build
+    echo "Building site" ;
+    COMPOSER_HOME=/var/cache/composer composer build ;
+    if [ $? != 0 ];then log_error_and_exit composer_build $?; fi ;
+)
+
+echo "[DONE] deploy apigility.org"
```
Comment (michalbundyra) [inline .deploy/deploy.sh]: the same here

File: .deploy/travis_deploy.sh [hunk 3]
```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+TOKEN=$1
+SITE=$2
+COMMIT=$3
+DATA='{"site":"'"${SITE}"'","commit":"'"${COMMIT}"'"}'
+
+curl -k \
+    --request POST \
+    --header "Authorization: Bearer ${TOKEN}" \
+    --header "Content-Type: application/x-zfweb-deploy+json" \
```
Comment (michalbundyra) [inline .deploy/travis_deploy.sh]: is zfweb-deploy correct?

File: composer.json [hunk 4]
```diff
@@ -0,0 +1,62 @@
+{
+    "name": "zfcampus/apigility.org",
```
Comment (michalbundyra) [inline composer.json]: update please

File: module/Application/view/application/video/index.phtml [hunk 5]
```diff
@@ -0,0 +1,16 @@
+<section class="features row">
+    <h1>Apigility Introduction</h1>
```
Comment (michalbundyra) [inline module/Application/view/application/video/index.phtml]: Should this be updated or it's fine as the video is about Apigility?

File: .deploy/deploy.sh (hunk 1 above)
Comment (weierophinney) [reply .deploy/deploy.sh]: This entire directory is obsolete with the move to platform.sh. I'll remove it in a subsequent patch.

File: module/Application/view/application/video/index.phtml (hunk 5 above)
Comment (weierophinney) [reply module/Application/view/application/video/index.phtml]: Updated in a subsequent patch.

**Pull request merged**